lib/application/pages/home/collections/update/update_collection_page.dart

```diff
@@ -173,3 +232,115 @@ class _MemosActionButton extends ConsumerWidget {
     );
   }
 }
+
+class _MemosReorderableList extends ConsumerWidget {
+  const _MemosReorderableList({
+    required this.memos,
+    required this.currentMemoIndex,
+    required this.onReorder,
+    this.onTap,
+  });
+
+  final List<MemoMetadata> memos;
+
+  /// Currently Memo being visualized when the list is opened.
+  ///
+  /// This will be highlighted in the list to indicate which Memo the user is seeing.
+  final int currentMemoIndex;
+
+  /// Triggers when a Memo is moved from `oldIndex` to `newIndex`.
+  final void Function(int oldIndex, int newIndex) onReorder;
+
+  /// Triggers when Memo at `index` is tapped.
+  final void Function(int index)? onTap;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final textTheme = Theme.of(context).textTheme;
+
+    final listHeader = Text('${strings.jumpTo}...', style: textTheme.subtitle1, textAlign: TextAlign.center);
+
+    return Theme(
+      // Overrides theme to remove canvasColor and shadowColor when dragging a Memo card
+      data: Theme.of(context).copyWith(canvasColor: Colors.transparent, shadowColor: Colors.transparent),
+      child: ReorderableListView.builder(
+        header: listHeader.withSymmetricalPadding(context, vertical: Spacing.medium),
+        itemCount: memos.length,
+        onReorder: (oldIndex, newIndex) {
+          // `ReorderableListView` bug. Waiting for a fix in https://github.com/flutter/flutter/issues/24786.
+          if (newIndex > oldIndex) {
+            newIndex -= 1;
+          }
+          onReorder(oldIndex, newIndex);
+        },
+        physics: const ClampingScrollPhysics(),
```

Comment:
Any reason for clamping these physics? Bouncing on iOS/Clamping on Android should be default, as to keep the minimal native experience feel.

Reply:
removed